getting-started/background/translate.go

```diff
@@ -80,6 +85,40 @@ func Translate(ctx context.Context, m PubSubMessage) error {
 			return fmt.Errorf("firestore.NewClient: %v", err)
 		}
 	}
+	return nil
+}
+
+// [END getting_started_background_translate_init]
+
+// [START getting_started_background_translate_string]
+
+// translateString translates text to lang, returning the translated text,
```

Comment:
nit: I'd find this easier to read with a list of return objects.

Reply:
Done (changed to a list).